src/Uno.Toolkit.UI/Behaviors/SelectorExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+using Windows.Foundation.Collections;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using PipsPager = Microsoft.UI.Xaml.Controls.PipsPager;
+#endif
+
+namespace Uno.Toolkit.UI;
+public static partial class SelectorExtensions
+{
+	/// <summary>
+	/// Backing property for the <see cref="PipsPager"/> that will be linked to the desired <see cref="Selector"/> control.
+	/// </summary>
+	public static DependencyProperty PipsPagerProperty { get; } =
+		DependencyProperty.RegisterAttached("PipsPager", typeof(PipsPager), typeof(SelectorExtensions), new PropertyMetadata(null, OnPipsPagerChanged));
+
+	static void OnPipsPagerChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
+	{
+		if (args.NewValue == args.OldValue || dependencyObject is not Selector flipView)
+			return;
+
+		var pipsPager = (PipsPager)args.NewValue;
+
+		var selectedIndexBinding = new Binding
+		{
+			Mode = BindingMode.TwoWay,
+			Source = flipView,
+			Path = new PropertyPath(nameof(flipView.SelectedIndex))
+		};
+
+		pipsPager.SetBinding(PipsPager.SelectedPageIndexProperty, selectedIndexBinding);
```

Comment:
I'm curious how this would work in comparison to just hooking up to the `SelectionChanged` event. In the case that the attached PipsPager is removed or changes, do we need to somehow track this binding and remove it from the old PipsPager? @Xiaoy312, should we just go with the event subscribe/unsubscribe route over using a Binding?

Reply:
@kazo0 using the event will add more work and complexity, because this is `TwoWay`, so we will need to know when the change happens on `PipsPager` and reflect that to `Selector` and when that happens on `Selector` and reflect that to `PipsPager`. Using the Binding abstract this extra complexity.

Reply:
ill allow it, but you need to clear the binding when NewValue is null

Reply:
@Xiaoy312 any particular reason? I tested and there's no memory leaks

Reply:
the setter should properly clear up any subscriptions for previous value
given the follow scenarios:
1. SetXyz(obj, something); SetXyz(obj, null);
2. SetXyz(obj, something); SetXyz(obj, somethingElse);

in case 1, the binding + sync method are not cleared/unsubscribed
in case 2, the binding (with something) is not clear